docs/strategy-customization.md

```diff
@@ -249,6 +249,22 @@ minimal_roi = {
 
 While technically not completely disabled, this would sell once the trade reaches 10000% Profit.
 
+To use times based on candles, the following snippet can be handy.
+This will allow you to change the ticket_interval, and ROI will be set as candles (e.g. after 3 candles ...)
```

Comment:
```suggestion
This will allow you to change the ticket_interval for the strategy, and ROI times will still be set as candles (e.g. after 3 candles ...)
```